app/celery/tasks.py

```diff
@@ -20,7 +20,7 @@
 
 @notify_celery.task(name="sanitise-and-upload-letter")
 def sanitise_and_upload_letter(notification_id, filename, allow_international_letters=False):
-    current_app.logger.info("Sanitising notification with id {}".format(notification_id))
+    current_app.logger.info("Sanitising notification with id %s", notification_id)
```

Comment:
This type of formatting is so hard to read :')

I read the convo in utils PR (by the way, your PR message links api PR, not the utils PR, not sure if that's on purpose), so I understand it's the standard formatting for interpolation logs, but my skin still crawls when I see it 😅